### PR TITLE
move: Deployment Previews into README

### PR DESCRIPTION
**Pull request by glasnt**
## Description

Fixes b/294650721

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: run/deployment-previews/README.md [hunk 1]
```diff
@@ -1,25 +1,522 @@
...
+is located within the [Python Docs Samples](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/run/deployment-previews).
+Instead of adding this script into your source code, you can optionally build this script
+into a container within your project and run that container as a step in your Cloud Build configuration.
+
+You can complete the following instructions using either
+[Cloud Shell](https://console.cloud.google.com/?cloudshell=true), or your local
+machine if you have installed and configured `git` and Google Cloud CLI. The
+instructions below show both methods.
+
+TODO(glasnt0: cleanup)
```
Comment (averikitsch) [inline run/deployment-previews/README.md:215]: Noting "TODO"

File: run/deployment-previews/README.md [hunk 2]
```diff
@@ -1,25 +1,522 @@
...
+    gcloud config set project $PROJECT_ID
+    ```
+
+    If using Cloud Shell, you may need to authorize Google Cloud CLI to make a
+    Google Cloud API call. Click <b>Authorize</b> to allow this action to proceed.
+* Create a new container image:
 
 ```
-docker run --rm deployment-previews
+git clone https://github.com/GoogleCloudPlatform/python-docs-samples
```
Comment (averikitsch) [inline run/deployment-previews/README.md:230]: These code blocks may look better indented

File: run/deployment-previews/README.md [hunk 3]
```diff
@@ -1,25 +1,522 @@
...
+    1. Enter the following details:
+      *   Name: `preview-deploy`
+      *   Event: Pull Request
+      *   Source Repository: "<var>GITHUB_USER_NAME</var>/helloworld-python"
+      *   Base branch: `^main$`
+      *   Comment control: Required except for owners and collaborators
+         -  As the owner of the repo, previews will be automatically built on pull requests you create.
+         -  If you want to allow anyone to preview their changes, read more about the [security implications](https://cloud.google.com/build/docs/cloud-build-service-account) of selecting "Not required".
+      *   Configuration: Cloud Build Configuration file
+      *   Cloud Build configuration file location: `cloudbuild-preview.yaml`
```
Comment (averikitsch) [inline run/deployment-previews/README.md:280]: Check if you are happy with how this is rendering in Markdown

File: run/deployment-previews/README.md [hunk 4]
```diff
@@ -1,25 +1,522 @@
...
+    </figure>
+
+    1. Make a change; for example, change "Hello" to "Greetings".
+    1. Select **Create a new branch for this commit and start a pull request**, then click **Propose change**.
+1. Create a [new pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) with this branch.
+
+  If the trigger is configured correctly, a new check is displayed soon after
+you create the pull request:
+
+    <figure id="github-pending-check">
```
Comment (averikitsch) [inline run/deployment-previews/README.md:305]: Figure not rendering. You may need to check the indentation or not have a newline between the figure and text

File: run/deployment-previews/README.md [hunk 5]
```diff
@@ -1,25 +1,522 @@
...
+  <figure id="preview-service">
+    <img
+      src="images/deployment-previews-preview-service.png"
+      alt="Screenshot of the preview deployed Cloud Run service, showing 'Greetings World!'." />
+  </figure>
+
+  Note that if you navigate to the original service URL, the original content
+is shown:
+
+    <figure id="deployed-service">
```
Comment (averikitsch) [inline run/deployment-previews/README.md:340]: Not rendering figure

File: run/deployment-previews/README.md [hunk 6]
```diff
@@ -1,25 +1,522 @@
...
+      <img
+        src="images/deployment-previews-deployed-service.png"
+        alt="Screenshot of the originally deployed Cloud Run service, still showing 'Hello World!'." />
+    </figure>
+
+
+1. [View the revision list for your service](https://cloud.google.com/run/docs/managing/revisions#viewing_the_list_of_revisions_for_a_service) to check the service state in Cloud Run: there are now two revisions serving traffic: the original and the preview:
```
Comment (averikitsch) [inline run/deployment-previews/README.md:347]: Rendering as a new list

File: run/deployment-previews/README.md (hunk 3 above)
Comment (glasnt) [reply run/deployment-previews/README.md]: Corrected ✨

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**